src/commandDetails/admin/ban.ts

```diff
@@ -0,0 +1,72 @@
+import { container } from '@sapphire/framework';
+import { Permissions, User, Message } from 'discord.js';
+import {
+  CodeyCommandDetails,
+  CodeyCommandOptionType,
+  SapphireMessageExecuteType,
+} from '../../codeyCommand';
+import { banUser } from '../../components/admin';
+import { vars } from '../../config';
+
+// Ban a user
+const banExecuteCommand: SapphireMessageExecuteType = async (client, messageFromUser, args) => {
+  if (!(<Readonly<Permissions>>messageFromUser.member?.permissions).has('ADMINISTRATOR')) {
+    return `You do not have permission to use this command.`;
+  }
+
+  const user = <User>args['user'];
+  if (!user) {
+    throw new Error('please enter a valid user mention or ID');
+  }
+
+  const reason = <string>args['reason'];
+  if (!reason) {
+    throw new Error('please enter a valid reason why you are banning the user.');
+  }
+
+  const days = <number>args['days'];
+
+  // Get the GuildMember object corresponding to the user in the guild
+  // This is needed because we can only ban GuildMembers, not Users
+  const guild = await client.guilds.fetch(vars.TARGET_GUILD_ID);
```

Comment:
Do you mind updating the description of `TARGET_GUILD_ID` in `SETUP.md`, since right now, it looks like it is only used for coffee chats?